package/yast2.spec

```diff
@@ -52,6 +52,8 @@ BuildRequires:  yast2-ycp-ui-bindings >= 3.2.0
 # these should be installed in the default build anyway
 BuildRequires:  rpm
 BuildRequires:  cpio
+# YaST rake tasks
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:yast-rake)
```

Comment:
NP: I'd just move the line up so the `BuildRequires:  rubygem(%{rb_default_ruby_abi}:X)` are together.